Classes/Domain/AbstractEventSourcedAggregateRoot.php

```diff
@@ -42,8 +42,8 @@
     public static function reconstituteFromEventStream(string $identifier, EventStream $stream)
     {
         $instance = new static($identifier);
-        foreach ($stream as $eventWithMetadata) {
-            $instance->apply($eventWithMetadata->getEvent());
+        foreach ($stream as $eventTransport) {
+            $instance->apply($eventTransport->getEvent());
```

Comment:
> Shouldn't rather tell what it is or what it contains?

Well, yes, that's what I mean more or less: What does an "event transport" contain.. Anyways, it's only used by the framework so probably no biggy.